### PR TITLE
Team Fedex++ post two

### PR DESCRIPTION
**Pull request by mansigoel**
2016-10-11-team-fedex++-signing-off 
Added second blog post by Team Fedex++

### CODE REVIEW

File: blog/_posts/2016-10-11-team-fedex++-signing-off.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+![Team PyDSA](/img/blog/2016/TeamFedex++pydsa.png)    
+*All images have been taken from Rails Girls Team App with permissions*   
+
+### **What we achieved** - Majorly Web App 
+- In the second half of the program, we focussed upon the frontend part of the WebApp of PyDSA.
+- We explored D3.js , tried some things in it and finally made the visualisation of Bubble Sort Algorithm and Graph Data Structure using it. 
+- Though the PyDSA App is still in development mode, stay tuned for it first official release :)
+
+![Bubble Sort visualization](/img/blog/2016/TeamFedex++sort.gif)
```
Comment (alicetragedy) [inline blog/_posts/2016-10-11-team-fedex++-signing-off.md:39]: this is missing image credits.

File: blog/_posts/2016-10-11-team-fedex++-signing-off.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+### **Experience working in open source** 
+- This was our first proper experience with the Open Source community.
+- We had primarily taken up the RGSoC project to get exposure to Open Source, to foray into this never ending space filled with excitement, to get to know how to code into an actual live project repo and the feeling when your PR gets merged. We learnt a lot about community bonding, for instance it is not about what you learn but what you return to the community, how even a small change or input from your side may impact someone’s life deeply.
+- We got to interact with many people as part of conferences (PyCon India) and some very inspirational ladies (PyLadies Pune Chapter)
+- It is always a challenge to handle your academics along with your passion when you are a student but through the summer program we realised the importance of being a part of the community and what the world of open source can teach you which you might never learn under the bounds of an institute.
+
+
+![Team PyDSA](/img/blog/2016/TeamFedex++pydsa.png)    
+*All images have been taken from Rails Girls Team App with permissions*
```
Comment (alicetragedy) [inline blog/_posts/2016-10-11-team-fedex++-signing-off.md]: Why is this credited with "taken from Rails Girls Team app"? If one of you took a screenshot of your hangout, please use your own name as a credit. Or was it taken from someone else's daily log in the teams app?

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hey @mansigoel, we need a little more than "image taken with proper permissions" as a credit. As explained in the [contributing guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images), if the image is from the internet, we need a website name and url. If you asked someone to take a picture of you with your phones or cameras, you need to at least write down as credit the name of your team (see past posts for examples) to explain that you are the "owners" of the pictures—basically in the off chance that we might want to reuse the pictures somehow, so that we can credit you accordingly :)

Another thing that might also be nice (though optional) would be to have a short description of the image underneath it, before the image credits (even though you have that in the alt text already, so it's fine the way it is, too)!
Please make these small changes, and we will be ready to merge!

**Comment by mansigoel** [comment]
Hi @alicetragedy , 
Thank you for the feedback :)
I have made the required changes. Kindly check if anything else is needed :)

File: blog/_posts/2016-10-11-team-fedex++-signing-off.md (hunk 2 above)
Comment (mansigoel) [reply blog/_posts/2016-10-11-team-fedex++-signing-off.md]: Actually we uploaded this screenshot on our daily logs so we thought we need to use it as a source as logs are public. But I have changed it now. Kindly check :)

**Review by alicetragedy — APPROVED** [review]
looks good, thank you for making the necessary changes so quickly. We'll publish it soon!

**Comment by mansigoel** [comment]
Thanks a lot. :D

File: blog/_posts/2016-10-11-team-fedex++-signing-off.md (hunk 2 above)
Comment (alicetragedy) [reply blog/_posts/2016-10-11-team-fedex++-signing-off.md]: no, don't worry about it it doesn't have to be so strict ;)

**Pull request merged**